src/imageio/format/pdf.c

```diff
@@ -675,7 +675,7 @@ void gui_init(dt_imageio_module_format_t *self)
 
   d->pages = dt_bauhaus_combobox_new(NULL);
   dt_bauhaus_widget_set_label(d->pages, NULL, _("TODO: pages"));
-  dt_bauhaus_combobox_add(d->pages, _("all"));
+  dt_bauhaus_combobox_add(d->pages, C_("pdf pages", "all"));
```

Comment:
"pdf pages" -> "count" ? we are counting something?

Reply:
From the code it didn't seem like it's counting something - rather it seems to select what kinds of pages should be added to the resulting PDF file: single images, contact sheet, or all of those. Of course I might've misunderstood, and since this feature is in "TODO" state, it doesn't actually even matter right now, as it won't be visible in the application interface.